### ISSUE TITLE
[Job via upwork or request]

### ISSUE THREAD

**Issue by Davetheraveuk** [butor/blackbird#300]
I have some requests that I would be more than happy to pay for:

Add the following following parameters:
MinimumProfit = 0.10 (in USD)
DemoBalance = 500.00 (in USD)

If "DemoMode=true" then we will use the "DemoBalance" parameter.

The "MinimumProfit" is self explantory. 
If we have a minimum profit of 0.10 cents then a trade is executed.

The log file will include our profit/loss and balance printout.

**Comment by Unbr8kbal** [comment]
Have you found somebody who is willing to do this? I'm interested in helping.

**Comment by timotheeguerinrb** [comment]
Some exchanges have a sandbox api in order to try with fake founds. I know that Gdax and Gemini exchanges offer that just by modifying the URL, but I haven't test this. The problem is that bitfinex doesn't have this, so it needs to be implemented...

**Comment by Davetheraveuk** [comment]
@Unbr8kbal 
No. I have found another github repo that has 99% of the functionality I am looking for.

**Comment by samholt** [comment]
@Davetheraveuk Can you share the link to the other github repo, for the community to then incorporate it into BlackBird ? :) And or your thoughts on future features the community should implement ?

**Comment by Davetheraveuk** [comment]
@samholt 

> Can you share the link to the other github repo, for the community to then incorporate it into BlackBird ? :)

Sure, here it is:
https://github.com/123noob/triangular

> And or your thoughts on future features the community should implement ?

Everything i have posted in the top post.
:)
